Dockerfile

```
@@ -3,164 +3,107 @@ FROM elixir:1.14-otp-25 AS builder
-    ELIXIR_ERL_OPTIONS="" \
     RELEASE_DISTRIBUTION=none \
     RELEASE_NODE=none \
-    ERL_EPMD_PORT=-1
+    ERL_EPMD_PORT=-1 \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US:en \
+    LC_ALL=en_US.UTF-8
 
-# Install runtime dependencies including Node.js
+# Install runtime dependencies and set up locale
 RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)